tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -1084,7 +1115,7 @@ private void computeOne(int n) {
                 System.out.println("writeAction.compute(" + n + ") - got fileinfo " + name + ": " + localeID);
             }
             try {
-                writeSummaryRow(output, choices, fileInfo.vc.problemCounter, name, localeID);
+                writeSummaryRow(output, choices, fileInfo.vc.problemCounter, name, localeID, level);
```

Comment:
Six params for writeSummaryRow is too many, made worse by this pr -- more techdebt to be addressed when moving all the presentation to the front end